partiql-planner/src/test/kotlin/org/partiql/planner/PlannerErrorReportingTests.kt

```diff
@@ -182,15 +181,19 @@ internal class PlannerErrorReportingTests {
                 assertOnProblemCount(0, 1)
             ),
             // Chained, demostrate missing trace.
+            // TODO: We currently don't have a good way to retain missing value information. The following test
+            //  should have 2 errors.
```

Comment:
Should this have 2 **warnings** rather than 2 **errors**? Also what would be the two warnings/errors?

Reply:
Might be worth tracking in a GH issue if there are more warnings/errors expected from the planner here.

Reply:
Updating the comment to give more context, however I'd say that this is inconsequential and not worthy of a GH issue to track. What we really care about are warnings/errors being populated in the first place. I'm honestly not entirely sure why these tests are strict on the number of warnings/errors.

If you debug a C program to fix an error, oftentimes recompiling it will show plenty more errors.